Formula/pygobject.rb

```diff
@@ -14,7 +14,7 @@ class Pygobject < Formula
 
   depends_on "pkg-config" => :build
   depends_on "glib"
-  depends_on "python"
+  depends_on "python@2"
```

Comment:
Does the test need tweaking here? As far as I can see in the bottles the new `python@2` formula ships no vanilla `python` executable. It should work either way as the bindings shouldn't be tied to one specific `python` and the system's `python` isn't going anywhere anytime soon, but the intention here _seems_ to be to test against the Python it was built against? Maybe? 😓.